doc/whats_new.rst

```diff
@@ -1753,7 +1753,7 @@ Changelog
    - SourceEstimate now supports algebra by `Alex Gramfort`_.
 
 API changes summary
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~
```

Comment:
it seems you have removed a few characters inadvertently. We try to avoid that, even if it may have some good side-effects.